service/matching/taskWriter.go

```diff
@@ -160,13 +161,15 @@ func (w *taskWriter) appendTask(
 	case w.appendCh <- req:
 		select {
 		case r := <-ch:
+			w.tlMgr.metricScope.RecordTimer(metrics.TaskWriteLatencyPerTaskQueue, time.Since(startTime))
```

Comment:
this will include the queuing time (task waiting in taskWriter's channel).

Reply:
exactly, the queuing time is what I'd like to emit. For per task latency, without queuing, we use use the persist matching create task latency.